crates/astria-bridge-withdrawer/src/metrics.rs

```diff
@@ -26,6 +29,27 @@ impl Metrics {
         describe_gauge!(CURRENT_NONCE, Unit::Count, "The current nonce");
         let current_nonce = gauge!(CURRENT_NONCE);
 
+        describe_counter!(
+            NONCE_FETCH_COUNT,
+            Unit::Count,
+            "The number of times a fetch nonce request to sequencer was made"
```

Comment:
This is a bit misleading, since it sounds like it also includes attempts where the fetch failed.  Maybe we should rephrase as something like "The number of times the nonce was fetched from the sequencer" (and we could move the call to `increment_nonce_fetch_count` to after the nonce has been fetched rather than before just to be accurate)